.github/workflows/macos_clang.yaml

```diff
@@ -25,7 +25,9 @@ jobs:
         run: git lfs checkout
 
       - name: Install vcpkg packages
-        # vcpkg does not compile with macos-clang, use default gcc :|
+        env:
+          CC: /usr/local/opt/llvm/bin/clang
+          CXX: /usr/local/opt/llvm/bin/clang++
```

Comment:
Hey, I've tried to resolve this issue and I came across this statement. It would seem that for now the only way to compile vcpkg binary is to use gcc. However, later on it is absolutely fine to compile & install packages using clang
https://github.com/microsoft/vcpkg/issues/3593#issuecomment-508283461